Project.toml

```diff
@@ -1,25 +1,25 @@
 name = "PlanetaryEphemeris"
 uuid = "d83715d0-7e5f-11e9-1a59-4137b20d8363"
 authors = ["Jorge A. Pérez Hernández", "Luis Benet"]
-version = "0.2.1"
+version = "0.3.0"
 
 [deps]
 AutoHashEquals = "15f4f7f2-30c1-5605-9d31-71845cf9641f"
 Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
 DelimitedFiles = "8bb1440f-4735-579b-a4ab-409b98df4dab"
-JLD = "4138dd39-2aa7-5051-a626-17a0bb65d9c8"
+JLD2 = "033835bb-8acc-5ee8-8aae-3f567f8a3819"
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
 Quadmath = "be4d8f0f-7fa4-5f49-b795-2f01399ab2dd"
+SnoopPrecompile = "66db9d55-30c0-4569-8b51-7e840670fc0c"
 TaylorIntegration = "92b13dbe-c966-51a2-8445-caca9f8a7d42"
 TaylorSeries = "6aa5eb33-94cf-58f4-a9d0-e4b2c4fc25ea"
 
 [compat]
 AutoHashEquals = "0.2"
-JLD = "0.12"
 Quadmath = "0.5"
-TaylorIntegration = "0.8"
-TaylorSeries = "0.12"
+TaylorIntegration = "0.11"
+TaylorSeries = "0.13"
```

Comment:
Another request: can you upgrade `TaylorSeries` to v0.14? It is in the process of being registered.

Reply:
Since `PlanetaryEphemeris` depends on  `TaylorIntegration`, the latter should update the `TaylorSeries` version first.

Reply:
Good catch! I'll do it during the day...

Reply:
I went ahead and updated TaylorIntegration.jl (see PerezHz/TaylorIntegration.jl#159), new version v0.11.1 should be ready in a bit

Reply:
I've updated `TaylorSeries` to v0.14.